resources/js/visual-editor/blocks/gallery/gallery.css

```
@@ -0,0 +1,518 @@
+}
+
+.wp-block-gallery.has-nested-images.aligncenter {
+    justify-content: center;
+}
+
+/* ---- Theme: figcaption styling ---- */
+
+.blocks-gallery-caption {
+    margin-top: 0.5em;
+    margin-bottom: 1em;
+    color: currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)